app/bundles/CampaignBundle/Executioner/InactiveExecutioner.php

```diff
@@ -197,7 +197,7 @@ public function validate($decisionId, ContactLimiter $limiter, OutputInterface $
      */
     private function checkCampaignIsPublished()
     {
-        if (!$this->decisions->count()) {
+        if (!$this->decisions || !$this->decisions->count()) {
```

Comment:
What is the reason for adding this condition along with the other places similar to this in the PR? These 
 variables should always be an ArrayCollection and thus `->count()` should always be available.

Reply:
Yep, this one must always return an ArrayCollection. I got a bit overprotective.

The others you saw can return void because they get results from repos (which can end up with no-result/exceptions). This was resulting in a exceptions where you didn't intend them to be thrown at the end of while loops or while data was in flux.